src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsauditapi/resource/AuditResourceTest.kt

```diff
@@ -143,16 +143,82 @@ class AuditResourceTest : NoQueueListenerIntegrationTest() {
     @ParameterizedTest
     @MethodSource("secureEndpoints")
     internal fun `satisfies the correct role and scope`(uri: String) {
+      doNothing().whenever(auditService).sendAuditEvent(any())
```

Comment:
is this line needed?  Isn't the default to do nothing anyway?

Reply:
Just taken a look at this since it seemed a bit odd.  Confused at first, but then realised it's an integration test not a unit test.  Normally we would mock out the dependency with
```
  @MockBean
  private lateinit var auditMessagingTemplate: QueueMessagingTemplate
```
however the template isn't exposed as a bean, instead `AmazonSQSAsync` is injected and the template constructed.  Would be better to expose it as a bean separately, that way the `NoQueueListenerIntegrationTest` could then include the mock bean and you wouldn't need the `doNothing` all over the place...

Reply:
Good idea. Will look at that in another PR